### ISSUE TITLE
Mapping Metadata to Schemas

### ISSUE THREAD

**Issue by ntallman** [psu-libraries/cho-req#99]
As a metadata manager, 
I need to be able to be able to map metadata fields to standard schemas (e.g., Qualified Dublin Core), 
so metadata can be aggregated into different systems and/or reused.

Linda B/Nathan T

**Comment by ntallman** [comment]
User should be able to view/download metadata record for an object serialized to various metadata schemas.

Dublin Core -- OAI-PMH for DPLA and Summon
VRA
MODS/METS

**Comment by ntallman** [comment]
Related: #21

**Comment by cam156** [comment]
On a per item basis allow the returned format to be one of the above formats

**Comment by cam156** [comment]
Allow input of translation file from our metadata to the schema.  We may need to validate against the xml schema we creating.

**Comment by cam156** [comment]
The data could be coming from multiple field in our metadata for different work types

**Comment by ntallman** [comment]
Dublin Core mapping is most critical. If necessary, VRA and MODS can be split into other user stories.

**Comment by cam156** [comment]
@ntallman can we assume the metadata will be coming from the fields in @lmballinger's base map to begin with?  https://psu.box.com/s/r7b17lkluuyc052om80gqi5i1l5mwnyo

We might be able to start this by assuming the data would exists in a json format from the system natively and then convert that to the other formats.  That would allow us to do some work before the system decision was made.

**Comment by ntallman** [comment]
@cam156 Yes, the metadata will pull from the base schema and be mapped to other schemas. @lmballinger's CHO base map should identify the mappings (source/target).

**Comment by cam156** [comment]
Give me everything, one collection, dates filter
Sessions and continuing sessions

1) Controller for handling requests and filters
2) Converting each object to the correct format
